starknet_py/class_hash.py

```diff
@@ -0,0 +1,124 @@
+import json
```

Comment:
Let's not have it in the main directory of the library.

Reply:
Any idea where to put it? 👁️ 👄 👁️